docs/project/breaking-change-process.md

```diff
@@ -0,0 +1,44 @@
+# Breaking change process
+
+Breaking changes make it difficult for people to adopt new versions of .NET. We should avoid making them. In the rare case that you believe a breaking change is unavoidable, you should follow the process outlined in this document. The intent of the process is to get feedback. This might result in previously unseen  opportunities to achieve your goals without breaks, clever ways to limit breaks to more niche scenarios, and (most importantly) better understand the scope of impact to .NET users.
```

Comment:
```suggestion
Breaking changes make it difficult for people to adopt new versions of .NET. We strive to avoid making them. In the rare case when a breaking change is unavoidable or the best option when all factors are considered, you should follow the process outlined in this document. The intent of the process is to get feedback. This might result in previously unseen  opportunities to achieve your goals without breaks, clever ways to limit breaks to more niche scenarios, and (most importantly) better understand the scope of impact to .NET users.
```